lebedigital/demonstrator_scripts/computation_volume_content.py

```diff
@@ -0,0 +1,185 @@
+from lebedigital.unit_registry import ureg
+import pytest
+
+
+def computation_volume_content(input_dic):
+    """
+    This is the function to compute volume contents based on mass or volume ratios
+
+    The input and output is wrapped by the python pint package: https://pint.readthedocs.io/
+    This requires units to be attached to the input values.
+
+    Parameters
+    ----------
+    input_dic : dic / pint units required converted to matching units
+        required:
+            - 'density_cem'
+            - 'density_water'
+            - 'wb_mass_ratio' : water to binder ratio water/(cement + substitute (e.g. slag or cemII))
+            - 'density_aggregates'
+            - 'aggregates_volume_fraction'
+            - 'density_sub' : density of the substitute (slag or cemII)
+            - 'sc_volume_fraction' : volume fraction of substitute to base cement
+        optional (otherwise assumed to be zero)
+             - 'plasticizer_volume_content'
+             - 'density_plasticizer'
+        
+    Returns
+    -------
+    output : dic / pint unit given
+             - 'water_vol_fraction'
+             - 'sub_vol_fraction'
+             - 'cem_vol_fraction'
+             - 'cem_vol_content'
+             - 'sub_vol_content'
+             - 'water_vol_content'
```

Comment:
What is the difference between vol fraction and vol content? Is the second one the amount with a unit for a m3 of concrete?

Reply:
The volume fraction is the the fraction of the volume, like 1/3. It does not have units or m^3/m^3.
Volume content is the mass for 1m^3 of mix.
I added a comment to explain.

Reply:
So it is not a volume content (because it is a mass), but essentially the mass per concrete volume - the one that is usually specified for the mix design.

Reply:
well, that depends on your definition/understanding of "volume content" :D.
It is the amount of X within a defined volume. The unit is mass/length^3. 
If you think this might be confusing and have an alternative, we can easily change the name.